core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4728,6 +4729,27 @@ public String toString() {
   //
   // FUSE integration related properties
   //
+  public static final PropertyKey FUSE_AUTH_POLICY_CLASS =
+      new Builder(Name.FUSE_AUTH_POLICY_CLASS)
+          .setDefaultValue("alluxio.fuse.auth.DefaultAuthPolicy")
+          .setDescription("The fuse auth policy class.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.IGNORE)
+          .setScope(Scope.CLIENT)
+          .build();
+  public static final PropertyKey FUSE_AUTH_POLICY_CUSTOM_USER =
+      new Builder(Name.FUSE_AUTH_POLICY_CUSTOM_USER)
+          .setDefaultValue("<FUSE_AUTH_POLICY_CUSTOM_USER>")
```

Comment:
if no default value, then don't setDefaultValue, or we can use `alluxio` as the default custom user and group, if users want to use CustomAuthPolicy, they should change to their desired user or group